newjersey/navigator.business.nj.gov
# Healthcare

## Description



### Ticket



https://www.pivotaltracker.com/story/show/186827948

### Approach



### Steps to Test
Enable license status check via dynamics API. Added a feature flag b/c we don't have good API data at the moment. Updated env templates. Extracted switch statement in TaskUrlSlug so it can be used in Task-All collection in CMS (enables License Task Previews). 

Request to confirm variables placed in the backend are where they should be.



### Notes



## Code author checklist
